### PR TITLE
[Tooltip]: added hide-after timeout for hiding tooltip

### PR DESCRIPTION
**Pull request by ryatziv**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by ryatziv** [comment]
Added an optional prop `hide-after` to hide the tooltip after a number of millis.

**Comment by ryatziv** [comment]
Hi, sorry to bug you, but I was wondering if there is something I missed, or if you had any questions about the PR?

**Comment by ryatziv** [comment]
@Leopoldthecoder hello, I was wondering if there was something I missed in the PR, or if I'm just waiting in the queue?  Don't mean to bug you.

**Comment by Leopoldthecoder** [comment]
Oh, sorry, I'll take a look soon and hopefully this will be included in 1.4.5

**Comment by ryatziv** [comment]
Thank you.  Also waiting for 6428.  Much appreciated.

File: packages/tooltip/src/main.js [hunk 1]
```diff
@@ -118,15 +123,26 @@ export default {
...
+      if (this.hideAfter > 0) {
+        this.timeoutPending = setTimeout(() => {
+          this.showPopper = false;
+        }, this.hideAfter);
+      }
     },
 
     handleClosePopper() {
       if (this.enterable && this.expectedState || this.manual) return;
-      clearTimeout(this.timeout);
```
Comment (Leopoldthecoder) [inline packages/tooltip/src/main.js]: I think `clearTimeout(this.timeout);` should not be removed.

Comment (ryatziv) [reply packages/tooltip/src/main.js]: You don't want to remove the timeout when the tooltip is closed?  Why not?

Comment (Leopoldthecoder) [reply packages/tooltip/src/main.js]: I want to remove the timeout when the tooltip is closed. This is exactly what this line does, so it should not be removed.

Comment (ryatziv) [reply packages/tooltip/src/main.js]: ok... was the comment directed to me?  I'm confused.

Comment (ryatziv) [reply packages/tooltip/src/main.js]: I see what you mean.  I removed that line.  The line is restored.

**Comment by Leopoldthecoder** [comment]
@ryatziv Thanks for contributing.

**Comment by ryatziv** [comment]
Thanks @Leopoldthecoder (and thanks for Element!)
Would you please look at https://github.com/ElemeFE/element/pull/6428?

**Pull request merged**